frontend/src/pages/Login.js

```diff
@@ -0,0 +1,87 @@
+import PageLayout from '../components/PageLayout';
+import styled from 'styled-components/macro';
+import Footer from '../components/Footer';
+import { useState } from 'react';
+import { loginUser } from '../services/loginService';
+import { Redirect } from 'react-router-dom';
+
+export default function Login({ setJwtToken, jwtToken }) {
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        if (!username && !password) {
+            return;
+        }
+        loginUser(username, password).then(setJwtToken);
```

Comment:
Error handling!

Reply:
done! while I might need some pointers for transitions later ^^'